ci-operator/config/open-cluster-management/insights-client/open-cluster-management-insights-client-release-2.3.yaml

```diff
@@ -21,7 +21,6 @@ images:
       - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
   to: insights-client
 promotion:
-  disabled: true
```

Comment:
Why this change?  We only promote from the main branch.

Reply:
The disabled label exists in the current release branch. The instructions said to remove that line from the previous release. https://github.com/open-cluster-management/cicd-docs/blob/main/prow/new_release.md#update-the-prow-config-file-for-the-previous-release-branch

Reply:
ok, let's follow the instructions, maybe I don't understand this field well enough.